### PR TITLE
Fix compiling with new versions of efivar

### PR DESCRIPTION
**Pull request by hughsie**
Fixes https://github.com/fwupd/fwupd/issues/4181

Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [X] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

File: plugins/uefi-capsule/fu-uefi-common.h [hunk 1]
```diff
@@ -9,6 +9,7 @@
 
 #include <fwupdplugin.h>
 
+#include <efivar/efivar-types.h>
```
Comment (superm1) [inline plugins/uefi-capsule/fu-uefi-common.h:12]: seems you can't include this header in all cases, it should be part of the ifdef?

Comment (hughsie) [reply plugins/uefi-capsule/fu-uefi-common.h]: I should have used the main header; fixed.

**Review by superm1 — APPROVED** [review]

**Comment by superm1** [comment]
I think backport to various older branches too after merging.

**Comment by hughsie** [comment]
Backported to 1_4_X, 1_5_X and 1_6_X

**Pull request merged**